src/Utilities/ToolLocationHelper.cs

```diff
@@ -1060,7 +1060,7 @@ private static string[] GetLegacyTargetPlatformReferences(string targetPlatformI
                 ErrorUtilities.DebugTraceMessage("GetLegacyTargetPlatformReferences", "Encountered exception trying to gather the platform references: {0}", e.Message);
             }
 
-            return Array.Empty<string>();
+            return [];
```

Comment:
Question on these substitutions: as I read the docs on collection expressions, `[]` isn't guaranteed to be `Array.Empty<T>`; it depends on static analysis. Should we leave these? Or are we guaranteed to fall into that case because it worked before?